plugins/org.jboss.tools.openshift.express.ui/src/org/jboss/tools/openshift/express/internal/ui/explorer/ExplorerExpressLabelProvider.java

```diff
@@ -35,7 +35,7 @@
  * @author Xavier Coulon
  * @author Andre Dietisheim
  */
-public class OpenShiftExplorerLabelProvider implements IStyledLabelProvider, ILabelProvider {
+public class ExplorerExpressLabelProvider implements IStyledLabelProvider, ILabelProvider {
```

Comment:
The v3 label provider is called OpenShiftExplorerLabelProvider, shouldnt this be called ExpressExplorerLabelProvider for consistency?